.github/workflows/lint.yml

```diff
@@ -0,0 +1,45 @@
+name: Check if code is formatted
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  run-npm-command:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup asdf
+        uses: ynab/asdf-action@v1.2
+        with:
+          version: 0.16.7
+
+      - name: Setup Node version
+        working-directory: mobile
+        run: |
+          asdf plugin add nodejs
+          asdf install nodejs
+          echo "node_version=$(asdf current nodejs | xargs | cut -d ' ' -f 6)" >> $GITHUB_ENV
+
+      - name: Set nodejs as global exec
+        run: |
+          asdf set -u nodejs ${{ env.node_version }}
+
+      - name: Install dependencies
+        working-directory: mobile
+        run: npm ci
+
+      - name: Run ts:check
+        working-directory: mobile
+        run: npm run ts:check
+
+      - name: Run lint
+        working-directory: mobile
+        run: npm run lint
+
+      - name: Run prettier:fix
+        working-directory: mobile
+        run: npx prettier . --single-quote --check
```

Comment:
Why did you put `--single-quote` here? This rule is in the prettier's config file.